### PR TITLE
Redirect /login, /logout, /register to /auth/<path>

### PR DESCRIPTION
**Pull request by hharchani**

### CODE REVIEW

File: src/app/controllers/static_page.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    public function login() {
+        $this->http_lib->redirect( base_url() . 'auth/login' );
+    }
+
+    public function logout() {
+        $this->http_lib->redirect( base_url() . 'auth/logout/' );
+    }
+
+    public function register() {
+        $this->http_lib->redirect( base_url() . 'auth/register/' );
```
Comment (nisargjhaveri) [inline src/app/controllers/static_page.php]: Redirect this to `auth/login/`?

**Comment by hharchani** [comment]
Oops! Typo updated.

File: src/app/controllers/static_page.php (hunk 1 above)
Comment (nisargjhaveri) [inline src/app/controllers/static_page.php]: Status?

**Comment by hharchani** [comment]
Updated.

File: src/app/routes.php [hunk 2]
```diff
@@ -5,10 +5,15 @@
...
-    "/trash/"   => "/jugaad/trash/",
-    "/jugaad/"  => "/jugaad/read/",
-    "/ajax/"    => "/ajax/",
-    "/auth/"    => "/auth/",
+    "/trash/"       => "/jugaad/trash/",
+    "/jugaad/"      => "/jugaad/read/",
+    "/ajax/"        => "/ajax/",
+    "/auth/"        => "/auth/",
+
+    "/login/"       => "/static_page/login/",
```
Comment (polybuildr) [inline src/app/routes.php:13]: We could also have `"/login/"       => "/auth/login/",`, we prefer this PR over that?

Comment (polybuildr) [inline src/app/routes.php:13]: @nisargjhaveri told me that this isn't really possible.

**Pull request merged**